compose/config/validation.py

```diff
@@ -44,6 +48,20 @@ def format_ports(instance):
     return True
 
 
+@FormatChecker.cls_checks(
+    format="environment")
```

Comment:
I had been trying to do something clever previously with calling the `log.warn` instead of an exception but it didn't pan out. So yep, I'll make it one line.